### PR TITLE
Youtube link for GDG Chapter added.

### PR DESCRIPTION
**Pull request by SaptakS**
Added the youtube link parameter in config file and added the code in MainController to generate the youtube channel link. Solves #61

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,6 +58,9 @@ <h2 hide-sm>
                 <a ng-if="mc.facebookLink" ng-href="{{ mc.facebookLink }}" target="_blank">
                     <md-icon class="socialIcon" md-font-set="fa fa-lg fa-facebook-square" aria-label="Facebook"></md-icon>
                 </a>
+                <a ng-if="mc.youtubeLink" ng-href="{{ mc.youtubeLink }}" target="_blank">
+                    <md-icon class="socialIcon" md-font-set="fa fa-lg fa-youtube-square" aria-label="Youtube"></md-icon>
```
Comment (Splaktar) [inline index.html]: This works. Crazy that the material icons set doesn't have a YouTube icon.

**Comment by Splaktar** [comment]
Thank you very much!

**Comment by Splaktar** [comment]
If you use the words 'Fixes #61' then it will automatically close that issue when the PR is merged. Looks like 'Solves #61' doesn't work.

**Comment by SaptakS** [comment]
Ok.. Does it work even if I comment it now?

**Comment by Splaktar** [comment]
@SaptakS nope. I closed it manually. No problem.

**Pull request merged**